### PR TITLE
add 'sendfile on' to nginx conf

### PR DESCRIPTION
**Pull request by cat-bro**
based on https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/roles/geerlingguy.nginx/templates/nginx.conf.j2, this is probably what it needs

### CODE REVIEW

File: templates/nginx/galaxy.j2 [hunk 1]
```diff
@@ -7,6 +7,7 @@ server {
     # Our log files will go here.
     access_log  /var/log/nginx/access.log;
     error_log   /var/log/nginx/error.log;
+    sendfile    on;
```
Comment (Slugger70) [inline templates/nginx/galaxy.j2:10]: Do we have the _x_accel_redirect stuff already in the nginx conf?

Comment (Slugger70) [reply templates/nginx/galaxy.j2:10]: https://docs.galaxyproject.org/en/master/admin/nginx.html?highlight=jbrowse#sending-files-with-nginx

Comment (Slugger70) [reply templates/nginx/galaxy.j2:10]: And the galaxy.yml setting also

Comment (cat-bro) [reply templates/nginx/galaxy.j2:10]: we have this in our group_vars/galaxyservers.yml but it's commented out some other options I was not sure if we were using 
    # nginx_x_accel_redirect_base: /_x_accel_redirect

we also had the block in nginx configs on galaxy qld

location /_x_accel_redirect/ {
    internal;
    alias /;
}

Comment (cat-bro) [reply templates/nginx/galaxy.j2:10]: I'll add these in.  Are these instead of of in addition to using 'sendfile on'?

Comment (Slugger70) [reply templates/nginx/galaxy.j2:10]: Maybe, lets leave both and see what happens

**Review by Slugger70 — APPROVED** [review]

**Pull request merged**